paper-checkbox.html

```diff
@@ -187,6 +187,7 @@
         white-space: normal;
         line-height: normal;
         color: var(--paper-checkbox-label-color, var(--primary-text-color));
+        width: calc(100% - var(--paper-checkbox-size) - var(--paper-checkbox-label-spacing));
```

Comment:
Actually, after reading the CSS values spec, this appears to be a bug in the implementation of `calc()` that happens to exist in all of Chrome / Firefox / Safari. (I haven't tried out Edge yet.) Filed as https://bugs.chromium.org/p/chromium/issues/detail?id=641556